documentation/en/cli-lotus-miner.md

```diff
@@ -77,13 +82,15 @@ USAGE:
    lotus-miner init restore [command options] [backupFile]
 
 OPTIONS:
-   --nosync                don't check full-node sync status (default: false)
-   --config value          config file (config.toml)
-   --storage-config value  storage paths config (storage.json)
-   --help, -h              show help
+   --nosync                       don't check full-node sync status (default: "false")
+   --config                       config file (config.toml) (default: "")
```

Comment:
6. don't show default where one isn't set - you'll need to figure out the rule for that (is it the zero value? then how come `false` shows up sometimes but not others? :shrug:

Reply:
Done.